defaults/main.yml

```diff
@@ -34,9 +34,14 @@ acme_sh_default_dns_sleep: 120
 acme_sh_default_extra_flags_issue: ""
 acme_sh_default_extra_flags_renew: ""
 acme_sh_default_extra_flags_install_cert: ""
+acme_sh_default_extra_flags_deploy_cert: ""
+
+acme_sh_default_deploy_env_vars: {}
+acme_sh_default_deploy_hook: ""
 
 acme_sh_default_install_cert_reloadcmd: "sudo service nginx reload"
 
+
```

Comment:
Can you remove this extra new line?

Reply:
Hi @nickjj
Sorry for the delay. I'll fix that and no worry about the rewriting the README if it isn't fitting your requirements, I'm not a full time english speaker ;-)

Reply:
It's ok. If you can copy the exact wording, punctuation and grammar from the other parts of the docs that would be great. But I see that might be tough if English isn't your primary language.